src/server/models/db/migrations/20230302210317_add_view_FirstPublishedOnList/migration.sql

```diff
@@ -2,7 +2,10 @@
 -- select distinct on (l.id), similar to group by, ensures only 1 entry appears per listId
 -- min(e.time) selects the earliest time a published event occurred (firstPublished)
 create view "FirstPublishedOnList" as (
-  select distinct on (l.id) "listId", min(e.time) over (partition by l.id) "firstPublished", e.id "eventId"
-  from (select time, type, id, "listItemId" from "Event" where type = 'PUBLISHED') e
-  inner join (select "listId", "id" from "ListItem") li ON e."listItemId" = li.id
-  inner join "List" l ON li."listId" = l.id);
+ select distinct on (l.id) "listId", min(e.time) over (partition by l.id) "firstPublished", e.id "eventId", l."nextAnnualReviewStartDate"
+ from (select "id", "nextAnnualReviewStartDate" from "List") l
```

Comment:
Although this query will work for sure, are these subqueries needed?  Can you not just query directly `from "List" l` and `inner join on "ListItem" li ON li."listId" = l.id`?  That could perhaps be the source of the slower performance that I wasn't expecting